docs/getting-started.md

```diff
@@ -1,21 +1,162 @@
-Hi, are you new to [Zulip](https://github.com/zulip/zulip)? If so, we recommend trying out our [web-client](https://chat.zulip.org) first to understand the concept of [streams/topics/PMs](https://zulip.com/help/about-streams-and-topics) in the world of Zulip. Just sending a message or two in [#test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream should help you get started.
+Hi, are you new to [Zulip](https://github.com/zulip/zulip)? If so, I recommend trying out our [web-client](https://chat.zulip.org) first to understand the concept of [streams/topics/PMs](https://zulip.com/help/about-streams-and-topics) in the world of Zulip. Just sending a message or two in [#test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream should help.
```

Comment:
Ah, I almost forgot—along with the link to CZO (chat.zulip.org), I think it'd be good to link to the [CZO community doc](https://zulip.readthedocs.io/en/latest/contributing/chat-zulip-org.html), which has lots of useful information that we'd like people to be aware of.

Reply:
Added the plug in the first paragraph